src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const employeeListItems = document.querySelectorAll('ul > li');
+
+const employees = Array.from(employeeListItems).map((item) => {
+  return {
+    name: item.textContent.trim(),
+    position: item.getAttribute('data-position'),
+    salary: Number(item.getAttribute('data-salary').replace(/[$,]/g, '')),
+    age: item.getAttribute('data-age'),
+  };
+});
+
+const sortList = function (list) {
+  list.sort((a, b) => b.salary - a.salary);
+};
+
+const getEmployees = function (list) {
+  return list;
+};
```

Comment:
The `getEmployees` function currently just returns the list it receives without any modification or additional processing. If this function is not needed, consider removing it to simplify the code. Alternatively, if it is intended to perform some operations on the list, you might want to implement those operations.